cluster-install/README.adoc

```diff
@@ -36,18 +36,20 @@ Kops needs a “state store” to store configuration information of the cluster
     aws s3api create-bucket --bucket kubernetes-aws-config
     # enable versioning and export
     aws s3api put-bucket-versioning \
-      --bucket kubernetes-aws-config 
-      --versioning-configuration\
-    Status=Enabled
+      --bucket kubernetes-aws-config \
+      --versioning-configuration \
+      Status=Enabled
     export KOPS_STATE_STORE=s3://kubernetes-aws-config
 
-== Create hosted zone on Route53
+== Create hosted zone on Route53 (optional)
```

Comment:
Why make this section optional? We should encourage to use Route 53, and then fall back to gossip-based cluster.

@dalbhanj wdyt?

Reply:
It's a lot of additional effort to setup DNS - we should provide both options which is what I've done here. There are 3 different cluster setups in the README - 2 of them using gossip, one using DNS. If I was installing this for non-prod environments I'd definitely want to start with a gossip based setup, and move to DNS later.

Reply:
Fair point, let's use gossip-based cluster for simplicity. Can you explain this instead of marking the DNS section optional?

Let's make that one change and then we merge, ok?

Reply:
Yip, no worries

Reply:
ok - readme updated and PR updated